evalml/automl/automl_search.py

```diff
@@ -101,10 +101,12 @@ def __init__(self,
         Arguments:
             problem_type (str or ProblemTypes): Choice of 'regression', 'binary', or 'multiclass', depending on the desired problem type.
 
-            objective (str, ObjectiveBase): The objective to optimize for. When set to auto, chooses:
-                LogLossBinary for binary classification problems,
-                LogLossMulticlass for multiclass classification problems, and
-                R2 for regression problems.
+            objective (str, ObjectiveBase): The objective to optimize for. Used to select and rank models, but not for optimizing each model during fit-time.
```

Comment:
not-blocking: I think it would be better to use `pipelines` instead of `model` to match the language in the user guide. Also, I think `propose` is better than `select` hehe

Reply:
LOL okie dokie doke, will update!